json_pointer.c

```diff
@@ -0,0 +1,242 @@
+/*
+ * $Id: json_pointer.c,v 1.20 2016/10/25 03:24:50 mclark Exp $
+ *
+ * Copyright (c) 2016 Metaparadigm Pte. Ltd.
```

Comment:
These copyrights are wrong.  If you wrote the code, your name should be on the copyright.